.github/workflows/build.yml

```diff
@@ -100,6 +100,10 @@ jobs:
         # This is required because it's being set to llvm/ otherwise
         run: unset LD_LIBRARY_PATH
 
+      - name: Allow perf_events to be accessed by non-root users
+        # TODO(vthakkar): Fix the bug in FDTransfer to handle this
```

Comment:
JFYI, will handle this todo in the follow-up PR. Also, I'm keeping the debugging logs in the test for now as they're useful.